gamestonk_terminal/common/newsapi_view.py

```diff
@@ -0,0 +1,48 @@
+""" News View """
+__docformat__ = "numpy"
+
+from datetime import datetime, timedelta
+import requests
+from gamestonk_terminal import config_terminal as cfg
+
+
+def news(term: str, num: int):
+    """Display news for a given title. [Source: NewsAPI]
+
+    Parameters
+    ----------
+    term : str
+        term to search on the news articles
+    num : int
+        number of articles to display
+    """
+    # TODO: Add argument to specify news source being used
+    # TODO: Add argument to specify date start to search for articles
+    s_from = (datetime.now() - timedelta(days=7)).strftime("%Y-%m-%d")
+
+    response = requests.get(
+        f"https://newsapi.org/v2/everything?q={term}&from={s_from}"
+        f"&sortBy=publishedAt&language=en&apiKey={cfg.API_NEWS_TOKEN}",
+    )
+
+    # Check that the API response was successful
+    if response.status_code != 200:
+        print("Invalid News API token\n")
```

Comment:
should print out the error code, since there can be other errors than authentication (which is probably the most likely tbf) 

`f"Error in request {response.status_code}.  Check News API token"`